src/lib/src/services/lib.service.ts

```diff
@@ -1,7 +1,5 @@
 import {Inject, Injectable, InjectionToken, NgZone} from '@angular/core';
-import {Observable} from 'rxjs/Observable';
-import {ReplaySubject} from 'rxjs/ReplaySubject';
-import {Subject} from 'rxjs/Subject';
+import {Observable, ReplaySubject, Subject} from 'rxjs';
```

Comment:
Importing dependencies from the `rxjs` root package leads to the entire package being included instead of just the needed functionality/types. Please make sure to import these dependencies from their respective files, as was the case before. If this is no longer possible please point me to the documentation stating this fact...

Reply:
@chancezeus good call, it looks like this is still possible, the path has just changed. I'll update this PR with these change.

Reply:
@chancezeus actually, it looks like what I have now is the preferred method and that tree-shaking tools work with it. Here's an article I found on it:
https://blog.angularindepth.com/rxjs-tslint-rules-for-version-6-d10e2482292d

Also a reference from the RXJS migration guide:
https://github.com/ReactiveX/rxjs/blob/master/MIGRATION.md#import-paths

Reply:
It looks like you are correct, I will test/review the PR and merge it as soon as I have some time, thanks for the great work and info